.github/workflows/ci-check-jacoco.yml

```
@@ -2,24 +2,27 @@ name: CI check JaCoCo code-coverage
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Setup Scala
-        uses: olafurpg/setup-scala@v10
+        uses: actions/checkout@v4
         with:
-          java-version: "adopt@1.8"
+          persist-credentials: false
+      - name: Set up JDK
+        uses: actions/setup-java@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)